message msg_game{
	required int32 id = 1;
	required string tipo = 2;
	optional int32 target = 3;
	optional int32 x = 4;
	optional int32 y = 5;
	optional string nombre = 6;
	optional string imagen = 7;
	optional int32 ancho_base = 8;
	optional int32 alto_base = 9;
	optional int32 fps = 10;
	optional int32 delay = 11;
	optional int32 total_frames_line = 12;
	optional int32 pixels_dimension = 13;
	optional string owner = 14;
	optional int32 health = 15;
	optional int32 strength = 16;
	optional float precition = 17;
	optional bool building = 18;
}
